### ISSUE TITLE
Add and remove links from connect card

### ISSUE THREAD

**Issue by andrewtavis** [activist-org/activist#154]
### Terms

- [X] I have searched [open and closed feature requests](https://github.com/activist-org/activist/issues?q=is%3Aissue+label%3Afeature)
- [X] I agree to follow activist's [Code of Conduct](https://github.com/activist-org/activist/blob/main/.github/CODE_OF_CONDUCT.md)

### Description

This issue would be to allow someone to add or remove links to the connections card as seen on [activist.org/organizations/tech-from-below/](https://activist.org/organizations/tech-from-below/). This issue is blocked by #153 where the new field component that would appear on `Add account` being pressed would be made. The goal would be that the add field component would allow users to input new fields, and further that by pressing the `X` icons next to accounts that they could be removed. We'll think about editing in the future :)

Changes for this issue would go within [frontend/components/Cards/CardsConnect.vue](https://github.com/activist-org/activist/blob/main/frontend/components/Cards/CardsConnect.vue) 😊

### Contribution

Happy to work on this or support someone who has interest!

**Comment by andrewtavis** [comment]
Now that #153 is done, this issue is now unblocked and the component for adding in a field can be found at [frontend/components/TooltipNewField.vue](https://github.com/activist-org/activist/blob/main/frontend/components/TooltipNewField.vue) :)

**Comment by andrewtavis** [comment]
Hey @messiuw! Nice to meet you and thanks for your interest in activist :) Happy to assign you as this should be a good issue for a first timer contributor 😊 Please let me know if you have any questions. Would be happy to discuss :)

**Comment by messiuw** [comment]
What is the intended behaviour of the "New account" button? Upon clicking the button, should there be an empty input field to the left (or right) of the button or should a small modal dialog pop up where the user can enter the details?

**Comment by andrewtavis** [comment]
There should be a pop up that happens that allows the user to enter in a new field. It was originally labeled as a tooltip, but I renamed the file and the component can now be found at [frontend/components/Popup/PopupNewField.vue](https://github.com/activist-org/activist/blob/main/frontend/components/Popup/PopupNewField.vue) :) For this leave `descriptionPrompt` blank as we just want the text input, not the text area. Let me know if you have more questions!

**Comment by messiuw** [comment]
Alright, thanks! Onto the next question ;) Where do you want the popup to be displayed? Inside the CardsConnect component (next to the "New Account" button) or floating above the card? Like here https://flowbite.com/docs/components/popover/#triggering when clicking the button? Sorry for asking so many questions...

**Comment by messiuw** [comment]
I have created a pull request for this issue. Let me know if you would like to discuss the changes or if something needs explanation! Let me know if there are other issues where help is needed.

**Comment by andrewtavis** [comment]
Thank you, @messiuw! There will definitely be more issues that need help 😊 I'm finishing up some issue creation for another project now and then will hop over to activist to start reviewing the PRs here :) Hope you're having a nice day!

**Comment by andrewtavis** [comment]
@messiuw, 9cc640c adds some minor changes to the code to have the popup above the button and add in a transition. Really appreciate you using Headless UI for this 😊 Great to keep with the current style :) Hope to work with you again in the future!

**Comment by messiuw** [comment]
@andrewtavis would be happy to keep contributing to this project! If you like you can assign a new issue to me :)

**Comment by andrewtavis** [comment]
@messiuw, you need to write in issues before I can assign them 🙃 Keep in touch and maybe join us in the development discussions of the [public Matrix chat rooms](https://matrix.to/#/#activist_community:matrix.org) if you haven’t already! We’ll be making more issues soon :) :)
